### PR TITLE
grants/admin: base setup

### PR DESCRIPTION
**Pull request by thelostone-mc**
- adds dummy edit to name & description 
- TODO: fire API to save data

### CODE REVIEW

File: app/assets/v2/js/grants/detail.js [hunk 1]
```diff
@@ -1,3 +1,33 @@
...
+  $(input).prop('disabled', false);
+  $(icon).toggleClass('fa-edit').toggleClass('fa-save');
+};
+
+const inlineSave = (input, icon) => {
+  $(input).removeClass('editable');
+  $(input).prop('readonly', true);
+  $(input).prop('disabled', true);
+  $(icon).toggleClass('fa-edit').toggleClass('fa-save');
+  // TODO : fire API
```
Comment (thelostone-mc) [inline app/assets/v2/js/grants/detail.js]: @mbeacom

Comment (mbeacom) [reply app/assets/v2/js/grants/detail.js]: @thelostone-mc You just want an endpoint to `POST` the edit to?

File: app/grants/templates/grants/detail.html [hunk 2]
```diff
@@ -41,7 +41,12 @@ <h1 class="grant__title font-title-xl mb-4">
                   <a href="{{ grant.reference_url }}" target="_blank">
                     <i class="fab fa-github font-title"></i>
                   </a>
-                <span>{{ grant.title }}</span>
+                <span>
+                  {% if is_admin %}
```
Comment (mbeacom) [inline app/grants/templates/grants/detail.html]: 🙌

File: app/assets/v2/js/grants/detail.js (hunk 1 above)
Comment (thelostone-mc) [reply app/assets/v2/js/grants/detail.js]: yup

**Pull request merged**